projects/azure/README.md

```diff
@@ -25,6 +25,15 @@ ci-operator --config ci-operator/config/openshift/openshift-azure/master.json --
 
 # Secret rotation
 
+We use 2 types of secrets. Both of them contains same data. 
```

Comment:
s/contains/contain/

Reply:
> Both of them contains same data. 

They don't actually contain the same data. More precisely the format differs. We should probably call out that the env based secret is what we use for deploying clusters in Azure (so it's a drop-in config file for Openshift) whereas the file based approach is what ci-operator expects in order to source it in its environment.

Reply:
Sorry, azure-purge is using the env based secret. Is it the same as azure.conf? If not disregard my previous comment.

Reply:
One is just `export FOO=BAR` in the file, second one is `--from-literal=foo=${BAR}`. Same credentials, same variables just format.